src/peergos/shared/user/fs/TagIndexingService.java

```diff
@@ -0,0 +1,95 @@
+package peergos.shared.user.fs;
+
+import peergos.shared.Crypto;
+import peergos.shared.NetworkAccess;
+import peergos.shared.storage.auth.Bat;
+import peergos.shared.user.*;
+import peergos.shared.user.fs.transaction.Transaction;
+import peergos.shared.user.fs.transaction.TransactionService;
+import peergos.shared.util.ProgressConsumer;
+import peergos.shared.util.Serialize;
+
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.concurrent.CompletableFuture;
+
+public class TagIndexingService {
+    private static ProgressConsumer<Long> VOID_PROGRESS = l -> {
+    };
+    public static final String TAGS_INDEX_FILENAME = ".tagsindex";
```

Comment:
❓ 
Just curious, any particular reason why it is public?

Reply:
Good point! Changed :)